01/implement-highest-rated.checkpoint.md

```diff
@@ -0,0 +1,81 @@
+# Implement Highest Rated
+
+<!-- prettier-ignore-start -->
+### !challenge
+* type: code-snippet
+* language: python3.6
+* id: 95b91049
+* title: PSE
+* points: 3
+### !question
+
+Imagine working on software that deals with restaurant reviews.
+
+Create a function named `get_highest_rated` that is responsible for finding the highest-rated restaurant.
+
+This function should take in a list of dictionaries named `restaurants` as a parameter. Each dictionary represents the data associated with a restaurant, including its rating. This function should have a return value of the restaurant with the highest rating.
+
+| Input (Parameters of the function)                                           | Expected Output (Return value of the function) |
+| ---------------------------------------------------------------------------- | ---------------------------------------------- |
+| `restaurants = [{"name": "Grillby's", "rating": 1}, {"name": "Crow's Nest", "rating": 5}]` | `{"name": "Crow's Nest", "rating": 5}`         |
+| `restaurants = [{"name": "Crow's Nest", "rating": 1}]`                                     | `{"name": "Crow's Nest", "rating": 1}`         |
+| `restaurants = []`                                                                         | `None`                                         |
```

Comment:
I could imagine this easily getting out of sync with the question text on the previous page. May be worth adding a comment above each of them reminding us to update the other if we change one?